kotti/resources.py

```diff
@@ -223,7 +224,7 @@ class Node(Base, ContainerMixin, PersistentACLMixin):
     type = Column(String(30), nullable=False)
     #: ID of the node's parent
     #: (:class:`sqlalchemy.types.Integer`)
-    parent_id = Column(ForeignKey('nodes.id'))
+    parent_id = Column(ForeignKey('nodes.id'), index=True)
```

Comment:
Please enlighten me: how can this cause the tests to fail on PostgreSQL (https://travis-ci.org/disko/Kotti/jobs/78022891#L4714)???